runtime/win32.c

```diff
@@ -1085,6 +1085,39 @@ CAMLexport int caml_win32_isatty(int fd)
   return 0;
 }
 
+#ifndef ENABLE_VIRTUAL_TERMINAL_PROCESSING
+#define ENABLE_VIRTUAL_TERMINAL_PROCESSING 0x4
```

Comment:
```suggestion
#define ENABLE_VIRTUAL_TERMINAL_PROCESSING 0x0004
```
Very much bikenitpickshedding, but that's the exact [documented](https://learn.microsoft.com/en-us/windows/console/setconsolemode#parameters) value.

Reply:
Committed, thanks.